pkg/routeagent_driver/handlers/ovn/vsctl/vsctl.go

```diff
@@ -0,0 +1,97 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsctl
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	"k8s.io/klog"
+)
+
+const ovsCommandTimeout = 15
+
+func vsctlCmd(parameters ...string) (output string, err error) {
+	allParameters := []string{fmt.Sprintf("--timeout=%d", ovsCommandTimeout)}
+	allParameters = append(allParameters, parameters...)
+
+	cmd := exec.Command("/usr/bin/ovs-vsctl", allParameters...)
+
+	klog.V(log.DEBUG).Infof("ovs-vsctl %v", allParameters)
```

Comment:
TRACE?

Reply:
Only happens at startup, but yeah let's move it to TRACE.